src/app/views/data/data.js

```diff
@@ -20,6 +20,9 @@ class DataViewController extends ADBMapController {
     this.$log.debug('soum code:', this.soumCode);
 
     this.soumData.geojson(this.soumCode).then(geojson => this._addGeoJSONLayer(geojson));
+    this.soumData.load(this.soumCode).then(soum => {
```

Comment:
Oh, I forgot about that. Does the linter complain because you're returning the result of the assignment and not an actual sensible value? I personally think that's an annoying linter error and would be ok with disabling it, but I can also see cases where it would be useful.